.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
-  - "6"
-  - "8"
-  - "10"
+  - "14"
+  - "16"
+  - "18"
+  - "19"
```

Comment:
I'm 90% sure that travis isn't going to run.  Assuming it doesn't, I'd be happy to add GitHub actions to the repo.